WDE.SmartScriptEditor/SmartData/actions.json

```diff
@@ -120,15 +129,20 @@
     },
     {
         "id": 7,
-        "name": "SMART_ACTION_ADD_QUEST",
+        "name": "SMART_ACTION_OFFER_QUEST",
         "parameters": [
             {
                 "type": "QuestParameter",
                 "name": "Quest ID",
                 "description": "Add quest, only for players"
+            },
+            {
+                "type": "BoolParameter",
+                "name": "DirectAdd",
+                "description": "True - quest is added to questlog instantly, false - Quest Window pops up"
             }
         ],
-        "description": "{target}: Add quest {pram1}",
+        "description": "{target}: {pram2value:choose(0):Add quest {pram1} to quest log|Offer quest {pram1}}",
```

Comment:
oops, sorry, yesterday I made a mistake here, it is the opposite: choose(0) - offer quest, otherwise - add to questlog

```
{target}: {pram2value:choose(0):Offer quest {pram1}|Add quest {pram1} to quest log}
```